CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.2.0-dev1
+
+
+### Enhancements
+
+* **Added migration for GitLab Source V2**
+* **Updated the expected output for GitLab Source V2**
```

Comment:
Don't need this comment since it's specific to the testing framework, not relevant to users.

Reply:
Done